auto_submit/lib/validations/ci_successful.dart

```diff
@@ -95,6 +104,7 @@ class CiSuccessful extends Validation {
     if (!Config.reposWithTreeStatus.contains(slug)) {
       return true;
     }
+    // TODO for repositories that are merging into a non default branch this should not matter.
```

Comment:
Nit: do we need this doc?

Reply:
Ah no. Good catch. This was before I had the solution here. I will remove it.